sencha-workspace/tests/siesta-4.0.6-lite/tests/jarvus-aggregrid/Aggregrid.js

```diff
@@ -5,55 +5,63 @@ StartTest(function(t){
 
 			var AggregridView = Ext.create('Jarvus.aggregrid.Aggregrid', {
 				renderTo: Ext.getBody(),
-        columnsStore: {
-            //type: 'store',
-            fields: ['id', 'fullName'],
-            data: [
-                {id: 1, fullName: 'Ali', rank: 'Pro'},
-                {id: 2, fullName: 'Chris'},
-                {id: 3, fullName: 'Ryon'},
-                {id: 4, fullName: 'Kevin'},
-                {id: 5, fullName: 'Christian'}
-            ]
-        },
-        columnHeaderTpl: [
-            '{fullName}',
-            '<tpl if="rank">',
-                ' ({rank})',
-            '</tpl>'
-        ],
-
-        rowsStore: {
-            //type: 'store',
-            fields: ['id', 'taskName'],
-            data: [
-                {id: 1, taskName: 'Task 1'},
-                {id: 2, taskName: 'Task 2'},
-                {
-                    id: 3, taskName: 'Task 3',
-                    rows: [
-                        { taskName: 'Task 3 - Subtask 1'},
-                        { taskName: 'Task 3 - Subtask 2'},
-                        { taskName: 'Task 3 - Subtask 3'}
-                    ]
-                },
-                {id: 4, taskName: 'Task 4'}
-            ]
-        },
-        rowHeaderField: 'taskName'
-		});
+		    columnsStore: {
+		        //type: 'store',
+		        fields: ['id', 'fullName'],
+		        data: [
+		            {id: 1, fullName: 'Ali', rank: 'Pro'},
+		            {id: 2, fullName: 'Chris'},
+		            {id: 3, fullName: 'Ryon'},
+		            {id: 4, fullName: 'Kevin'},
+		            {id: 5, fullName: 'Christian'}
+		        ]
+		    },
+		    columnHeaderTpl: [
+		        '{fullName}',
+		        '<tpl if="rank">',
+		            ' ({rank})',
+		        '</tpl>'
+		    ],
+
+		    rowsStore: {
+		        //type: 'store',
+		        fields: ['id', 'taskName'],
+		        data: [
+		            {id: 1, taskName: 'Task 1'},
+		            {id: 2, taskName: 'Task 2'},
+		            {
+		                id: 3, taskName: 'Task 3',
+		                rows: [
+		                    { taskName: 'Task 3 - Subtask 1'},
+		                    { taskName: 'Task 3 - Subtask 2'},
+		                    { taskName: 'Task 3 - Subtask 3'}
+		                ]
+		            },
+		            {id: 4, taskName: 'Task 4'}
+		        ]
+		    },
+		    rowHeaderField: 'taskName'
+			});
+
+			t.firesOnce(AggregridView, 'refresh', '\'refresh\' event fired once after updateColumnsStore');
+
+			AggregridView.updateColumnsStore();
+
+			t.firesOnce(AggregridView, 'refresh', '\'refresh\' event fired once after updateRowsStore');
+
+			AggregridView.updateRowsStore();
```

Comment:
@aliyeysides the .updateFoo and .applyFoo functions generated for configs should almost _never_ be called directly.

If you want to test it you should be calling setColumnsStore/setRowsStore etc